website/pages/docs/builders/virtualbox/iso.mdx

```diff
@@ -230,41 +248,30 @@ uploaded is controllable by `guest_additions_path`, and defaults to
 "VBoxGuestAdditions.iso". Without an absolute path, it is uploaded to the home
 directory of the SSH user.
 
-## VBoxManage Commands
```

Comment:
This is a weird diff! I move `VBoxManage Commands` to a partial and added to the VBoxManage Configuration section at line 156 above.